docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -368,7 +368,44 @@ While developing you can also start adding tests one by one and run them with th
 
 === With Curl or Browser
 
-When you application is running in production mode you can test it with `curl` or your favourite browser.
+First, start a PostgreSQL server:
+
+[source,bash]
+----
+docker run --rm=true --name security-getting-started -e POSTGRES_USER=quarkus \
+           -e POSTGRES_PASSWORD=quarkus -e POSTGRES_DB=elytron_security_jpa \
```

Comment:
In `application.properties` we mention:

```
%prod.quarkus.datasource.jdbc.url=jdbc:postgresql:security_jpa
```

So it would be good to have these two aligned

Reply:
Good catch, thanks

Reply:
@geoand the actual quickstart uses `elytron_jpa_security` so I've fixed the doc itself to refer `elytron_jpa_security`, thanks